web/modules/contrib/auto_entitylabel/config/schema/auto_entitylabel.schema.yml

```diff
@@ -0,0 +1,16 @@
+auto_entitylabel.settings.*:
```

Comment:
Let's include these for now, but I suspect we can get rid of them once we sanitise the web/modules/contrib folder.

Reply:
Ya, figured i'd rather just include them in case for now.